modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/search/test/DDMFormInstanceRecordSearchTest.java

```diff
@@ -149,6 +149,88 @@ public void testExactPhrase() throws Exception {
 		assertSearch("Bloggs", 2);
 	}
 
+	@Test
+	public void testMultipleLanguageEntries() throws Exception {
```

Comment:
Can you rename testSearchLocalizedDDMFormInstanceRecord?

You should prioritize the use of the entity's names.